Content.Client/Stylesheets/Sheetlets/PanelContainerSheetlet.cs

```diff
@@ -0,0 +1,31 @@
+using Content.Client.Stylesheets.SheetletConfigs;
+using Content.Client.Stylesheets.Stylesheets;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using static Content.Client.Stylesheets.StylesheetHelpers;
+
+namespace Content.Client.Stylesheets.Sheetlets;
+
+[CommonSheetlet]
+public sealed class PanelContainerSheetlet<T> : Sheetlet<T> where T : PalettedStylesheet, IButtonConfig
```

Comment:
we already have a `PanelSheetlet` that you can add your rules to for this element